client/src/styles/components/_quiz.scss

```diff
@@ -179,3 +179,20 @@ $timer-size: 100px;
     height: 100%;
   }
 }
+
+/* ---------- DuelResults ---------- */
+
+.duel-results {
```

Comment:
Why keeping the button format?
It's a good idea to change the text to "win / lose" but having this big red or green banner is not great... It makes a big color block

Reply:
I quickly prototyped it as I was thinking about using a new _Pilette_ design to illustrate this part of the app. For the moment, I have applied your suggestions (and i looks much better :smile:)

<details>
  <summary></summary>
  The button is gone but something has replaced it! Grrrrrr
</details>

![image](https://user-images.githubusercontent.com/45119518/107712804-53335a00-6cca-11eb-8f78-25214029c861.png)
![image](https://user-images.githubusercontent.com/45119518/107712814-562e4a80-6cca-11eb-9cb5-b397d3eb3cb0.png)